src/wizard/dotnet/profiling-onboarding/dotnet/2.configure-performance.md

```diff
@@ -0,0 +1,37 @@
+---
+name: .NET
+doc_link: https://docs.sentry.io/platforms/dotnet/profiling/
+support_level: alpha
+type: language
+---
+
+#### Configure Performance
+
+Sentry’s performance monitoring product has to be enabled in order for Profiling to work. To enable performance monitoring in the SDK:
+
+```csharp
+using Sentry;
+
+SentrySdk.Init(options =>
+{
+    // A Sentry Data Source Name (DSN) is required.
+    // See https://docs.sentry.io/product/sentry-basics/dsn-explainer/
+    // You can set it in the SENTRY_DSN environment variable, or you can set it in code here.
+    options.Dsn = "___PUBLIC_DSN___";
+
+    // When debug is enabled, the Sentry client will emit detailed debugging information to the console.
+    // This might be helpful, or might interfere with the normal operation of your application.
+    // We enable it here for demonstration purposes when first trying Sentry.
+    // You shouldn't do this in your applications unless you're troubleshooting issues with Sentry.
+    options.Debug = true;
+
+    // This option is recommended. It enables Sentry's "Release Health" feature.
+    options.AutoSessionTracking = true;
+
+    // This option will enable Sentry's tracing features. You still need to start transactions and spans.
+    options.EnableTracing = true;
```

Comment:
This onboarding shows up in-product, so folks already have Sentry initialized, we can skip these additional options I think and only point to perf + profiling:

```suggestion
```